cmd/gitbase-web/main.go

```diff
@@ -56,14 +57,35 @@ func (c *ServeCommand) Execute(args []string) error {
 	static := handler.NewStatic("build/public", c.ServerURL, c.SelectLimit, c.FooterHTML)
 
 	// start the router
-	router := server.Router(logger, static, version, db, c.BblfshServerURL)
-	logger.Infof("listening on %s:%d", c.Host, c.Port)
-	err = http.ListenAndServe(fmt.Sprintf("%s:%d", c.Host, c.Port), router)
-	logger.Fatal(err)
+	router := server.Router(logrus.StandardLogger(), static, version, db, c.BblfshServerURL)
```

Comment:
We should consider `StandardLogger()` for go-log for this kind of use case...

Reply:
`StandardLogger` in go-log will return `logrus.Logger`? I'm not sure it's a good idea.

We pass logrus here because the router has request logging middleware that expects logrus interface.

Reply:
I have just realized that https://github.com/pressly/lg is deprecated, so we may not want to support this exact use case

Reply:
@smacker  I thought StandardLogger was meant to return a standard library logger...